### DISCUSSION TITLE
macOS version?

### DISCUSSION THREAD

**Discussion by AlexMihai1126** [deepskystacker/DSS#215] [Ideas]
Hello,
I am a huge fan of the software and I was wondering if there will ever be a mac os version available.

**Comment by perdrix52** [comment]
Strongly suggest you join the mailing list at https://groups.io/g/DeepSkyStacker

A Linux version is planned for early/mid next year, a MacOS version will follow at later time (I 'm pedalling as fast as I can).

**Reply by jamesfidell** [reply]
I believe that it should be possible to get to a point where almost exactly the same code works for Linux and MacOS with just occasional differences for pathnames, though getting the intrinsics to build and work could be quite a lot of fun.

**Comment by perdrix52** [comment]
ARM AVX intrinsics are not quite the same as the x64 Intel/AMD AVX, so that will take time
